src/query/atomWithInfiniteQuery.ts

```diff
@@ -5,37 +5,49 @@ import type {
   InitialDataFunction,
   QueryKey,
   QueryObserverResult,
-  RefetchOptions,
-  RefetchQueryFilters,
 } from 'react-query'
 import { atom } from 'jotai'
 import type { WritableAtom } from 'jotai'
 import { queryClientAtom } from './queryClientAtom'
 import { CreateQueryOptions, GetQueryClient } from './types'
 
-export type AtomWithInfiniteQueryAction<TQueryFnData> =
-  | ({ type: 'refetch' } & Partial<
```

Comment:
I started think that we need `refetch` action to recover from errors. Can't we keep it?